neurom/stats.py

```diff
@@ -176,3 +176,26 @@ def compare_two(data1, data2, test=StatTests.ks):
     Stats = namedtuple('Stats', ['dist', 'pvalue'])
 
     return Stats(*results)
+
+
+def total_score(paired_dats, p=2, test=StatTests.ks):
```

Comment:
Yes, it is important for p to be an input parameter because we need to test how the scores are modified according to p (and we don't have an optimal p to propose before we try many different ones).